script.js

```diff
@@ -211,15 +195,8 @@ class libraryCheckout{
   
   
```

Comment:
remove empty space before closing bracket, check space indendation on line 191 and 192 and allign them

Reply:
done